assets/src/edit-story/app/index.js

```diff
@@ -72,4 +75,5 @@ export {
 	useStory,
 	useConfig,
 	useFont,
+	useMedia,
```

Comment:
Do we really want to re-export it here? IMHO it's more natural to access it directly by `import { useMedia} from '..../media'`.

Reply:
I like exposing only a "limited" interface from importing app directly, but I'm not strongly against direct imports either.

Reply:
I mean, a similar argument can be made for font here. I think we shouldn't discuss this here.

Reply:
Let's run with this for now. I didn't notice initially the other few there. We can come back to it later.